path_login.go

```diff
@@ -95,11 +95,26 @@ func (b *kubeAuthBackend) pathLogin(ctx context.Context, req *logical.Request, d
 		return nil, errors.New("could not load backend configuration")
 	}
 
+	if err := validateAliasNameSource(role.AliasNameSource); err != nil {
+		b.Logger().Error(err.Error())
+		return nil, err
+	}
+
 	serviceAccount, err := b.parseAndValidateJWT(jwtStr, role, config)
 	if err != nil {
 		return nil, err
 	}
 
+	var aliasName string
+	switch role.AliasNameSource {
+	case aliasNameSourceSAToken:
+		aliasName = serviceAccount.UID
```

Comment:
Good catch! Fixed with https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/110/commits/23ce29fe0a96069475e173c17039b39ab8b7e18c

I will see if there is a better way to avoid this  confusion going forward.